docs/rules/jsx-props-no-spread-multi.md

```diff
@@ -0,0 +1,27 @@
+# Disallow JSX prop spreading the same expression multiple times (`react/jsx-props-no-spread-multi`)
+
+💼 This rule is enabled in the ☑️ `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs).
+
+<!-- end auto-generated rule header -->
+
+Enforces that any unique express is only spread once. Generally spreading the same expression twice is an indicator of a mistake since any attribute between the spreads may be overridden when
+the intent was not to. Even when that is not the case this will lead to unnecessary computations to be performed.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```jsx
+<App {...props} myAttr='1' {...props} />
```

Comment:
TODO: Should I add more expression examples eg. function calls?

Reply:
More examples, and more tests, are always good.